data_set/taskset/rewrites.csv

```diff
@@ -0,0 +1,17 @@
+Id, Rewrite (* Means that the task has been adapted for the context of the experiment.)
```

Comment:
nit: same as above. I suppose just being consistent would be fine (space after all commas or no space)